### PR TITLE
chore: [IAI-223] Pictograms and Illustrations' inventory

### PR DESCRIPTION
**Pull request by dmnplb**
### Preview
https://user-images.githubusercontent.com/1255491/183384274-3340d5a3-6709-4abd-ab2a-a8508584df3e.mp4

## Short description
This PR adds all the pictograms found in the codebase to UI Showroom, a first step towards a centralized management of the vector assets. It also adds a basic inventory of Illustrations.

## List of changes proposed in this pull request
- **UI Showroom**: Add the new `PictogramBox`, a temporary view component that show with a small label if the asset is managed as vector or raster asset.
- **UI Showroom**: Add the new `IllustrationBox` component

**Please note**: all these components used in the UI Showroom are temporary and consciously written without following best practices. 

## Trivia
This PR also updates the README file with local reference to the app icon and updated retina-ready store badges (both iOS and Android)

## How to test
* Start the IO app with `yarn run-ios`
* Login to the app
* Go to the _Profile → UI Showroom_

### CODE REVIEW

File: ts/screens/showroom/core/IllustrationsShowroom.tsx [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+const renderRasterImage = (image: ImageSourcePropType) => (
+  <NativeImage
+    source={image}
+    resizeMode={"contain"}
+    style={styles.image}
+    testID={"rasterImage"}
+  />
+);
+
+const IllustrationBox = (props: IllustrationBoxProps) => (
```
Comment (CrisTofani) [inline ts/screens/showroom/core/IllustrationsShowroom.tsx:145]: What's the difference between `IllustrationBox` and `PictogramBox` can both components be summarized into one common and standalone component?

File: ts/screens/showroom/core/PictogramsShowroom.tsx [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+const renderRasterImage = (image: ImageSourcePropType) => (
+  <NativeImage
+    source={image}
+    resizeMode={"contain"}
+    style={styles.image}
+    testID={"rasterImage"}
+  />
+);
+
+const PictogramBox = (props: PictogramBoxProps) => (
```
Comment (CrisTofani) [inline ts/screens/showroom/core/PictogramsShowroom.tsx:359]: same as above comment

**Review by CrisTofani — CHANGES_REQUESTED** [review]
PR LGTM, just one suggestion in order to simplify the code wrapper component for illustration and pictogram

File: ts/screens/showroom/core/IllustrationsShowroom.tsx (hunk 1 above)
Comment (dmnplb) [reply ts/screens/showroom/core/IllustrationsShowroom.tsx]: You're right. I needed something quick and dirty to view both assets at first. Now I've added a new component AssetViewerBox that replaces both.

**Review by CrisTofani — APPROVED** [review]

**Pull request merged**